src/mcode.ts

```diff
@@ -611,12 +675,60 @@ export class ExtractedMCODE {
 
   // Surgical Procedures
   getSurgicalProcedureValue(): string[] {
-    const surgicalValues:string[] = [];
-    // TODO - fill in with Surgical Procedures.
+
+    if(this.cancerRelatedSurgicalProcedure == null){
```

Comment:
If values are not present, does TJ expect them to be empty (like this -- an empty array), or set to null/undefined? 

Just for consistency!

Reply:
Yes, Trialjectory wants all empty send objects to have empty arrays for array values and null for single string values... Inconsistent, yeah, but it's what they want.